tensorflow/compiler/xla/service/gpu/gemm_rewriter.cc

```diff
@@ -1103,7 +1123,20 @@ class GemmRewriterVisitor : public DfsHloRewriteVisitor {
 
     std::vector<HloInstruction *> operands(gemm->operands().begin(),
                                            gemm->operands().end());
-    operands.insert(operands.begin() + 2, MaybeConstantFoldBias(bias));
+    HloInstruction *maybe_constant_folded_bias = MaybeConstantFoldBias(bias);
+    if (slice && bitcast) {
```

Comment:
I think we should create a bitcast even if there is no slice. As long as there is a bitcast, we need to bitcast the bias.